allauth/locale/vi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1092 @@
+msgstr "Không thể sử dụng tên đăng nhập này. Vui lòng chọn tên khác."
+
+#: account/adapter.py:57
+msgid "Too many failed login attempts. Try again later."
+msgstr "Đăng nhập thất bại nhiều lần. Vui lòng thử lại sau."
+
+#: account/adapter.py:59
+msgid "A user is already registered with this email address."
+msgstr "Địa chỉ email đã được đăng kí."
+
+#: account/adapter.py:60
+msgid "Incorrect password."
+msgstr "Sai mật khẩu."
+
+#: account/adapter.py:344
+#, python-brace-format
+msgid "Password must be a minimum of {0} characters."
+msgstr "Mật khẩu phải có ít nhất {0} kí tự."
+
+#: account/adapter.py:731
+msgid "Use your password"
+msgstr ""
+
+#: account/adapter.py:741
+msgid "Use your authenticator app"
+msgstr ""
+
+#: account/admin.py:23
+msgid "Mark selected email addresses as verified"
+msgstr "Đánh dấu địa chỉ email đã xác thực"
+
+#: account/apps.py:9
+msgid "Accounts"
+msgstr ""
+
+#: account/forms.py:60 account/forms.py:449
+msgid "You must type the same password each time."
+msgstr "Mật khẩu không khớp."
+
+#: account/forms.py:92 account/forms.py:412 account/forms.py:552
+#: account/forms.py:688
+msgid "Password"
+msgstr "Mật khẩu"
+
+#: account/forms.py:93
+msgid "Remember Me"
+msgstr "Ghi nhớ đăng nhập"
+
+#: account/forms.py:97
+msgid "This account is currently inactive."
+msgstr "asdf Tài khoản hiện không hoạt động."
+
+#: account/forms.py:99
+msgid "The email address and/or password you specified are not correct."
+msgstr "Sai email hoặc mật khẩu."
+
+#: account/forms.py:102
+msgid "The username and/or password you specified are not correct."
+msgstr "Sai tên đăng nhập hoặc mật khẩu."
+
+#: account/forms.py:112 account/forms.py:287 account/forms.py:477
+#: account/forms.py:572
+msgid "Email address"
+msgstr "Địa chỉ email"
+
+#: account/forms.py:116 account/forms.py:325 account/forms.py:474
+#: account/forms.py:567
+msgid "Email"
+msgstr "Email"
+
+#: account/forms.py:119 account/forms.py:122 account/forms.py:277
+#: account/forms.py:280
+msgid "Username"
+msgstr "Tên đăng nhập"
+
+#: account/forms.py:132
+msgid "Username or email"
+msgstr "Tên đăng nhập hoặc email"
+
+#: account/forms.py:135
+msgctxt "field label"
+msgid "Login"
+msgstr "Đăng nhập"
+
+#: account/forms.py:146
+msgid "Forgot your password?"
+msgstr "Quên mật khẩu?"
+
+#: account/forms.py:316
+msgid "Email (again)"
+msgstr "Nhập lại email"
+
+#: account/forms.py:320
+msgid "Email address confirmation"
+msgstr ""
+
+#: account/forms.py:328
+msgid "Email (optional)"
+msgstr ""
+
+#: account/forms.py:383
+msgid "You must type the same email each time."
+msgstr "Email không khớp"
+
+#: account/forms.py:418 account/forms.py:555
+msgid "Password (again)"
+msgstr "Nhập lại mật khẩu"
+
+#: account/forms.py:489
+msgid "This email address is already associated with this account."
+msgstr "Địa chỉ email đã được liên kết với tài khoản hiện tại."
+
+#: account/forms.py:491
+#, python-format
+msgid "You cannot add more than %d email addresses."
+msgstr ""
```

Comment:
There a quite some translations that are still to go. Can you look into those?

Reply:
That's the plan. Not sure if I have time for that now. Do you mind keeping this pr until I'm done?